### DISCUSSION TITLE
Is there a way to clone a page in Oqtane

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#4749] [General]
Is there a way to clone a page and all the modules on a page and create a new page with all the modules laid out in the same panes?

**Comment by sbwalker** [comment]
This feature does not exist in the Oqtane Admin interface. And the main question is if you are expecting the modules to be "cloned" by reference or by value. You could certainly create a custom module to implement this functionality as the APIs all exist already. However if you expect to clone by value then you need to keep in mind that it depends on whether a module implements the IPortable interface or not.

**Reply by vnetonline** [reply]
By value so the contents of the module can be changed

**Comment by markdav-is** [comment]
I'm not sure what you are trying to accomplish, but you can script the creation of pages using page migrations

**Comment by vnetonline** [comment]
I am trying to give a website administrator the ability to replicate a page so they don't have to add modules individually and populate the styling. Just modify the content within the modules.
